docs/topics/settings.rst

```diff
@@ -604,6 +604,21 @@ If you want to disable it set to 0.
 
     This feature needs Twisted >= 11.1.
 
+.. setting:: DOWNLOAD_ALLOW_DATALOSS
+
+DOWNLOAD_ALLOW_DATALOSS
```

Comment:
This `_DataLoss` reason which Twisted returns doesn't necessarily means data loss in practice, right? If it can mean something else then there is no reason to follow this naming convention and call it 'data loss', we can do better. That's why I tried to avoid it in these name ideas. Other than that `DOWNLOAD_FAIL_ON_DATALOSS` sounds fine, it is not ambiguous.